Source/QuantityTypes.Tests/QuantityTypes.Tests.csproj

```diff
@@ -89,7 +90,9 @@
       <Name>QuantityTypes</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />

```

Comment:
All the credit goes to Visual Studio 2013, I've done nothing ;-)
Thanks for pointing us at the explanation.